### PR TITLE
Update Magento_Backup module ReadMe

### PR DESCRIPTION
**Pull request by diazwatson**
### Description (*)

Updates ReadMe for the Magento_Backup module.

### Fixed Issues (if relevant)

1. magento/devdocs https://github.com/magento/devdocs/issues/4451: README content for modules - work in M2 Code

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Backup/README.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
-The Backup module allows administrators to perform backups and rollbacks. Types of backups include system, database and media backups. This module relies on the Cron module to schedule backups.
+# Magento_Backup module
 
-This module does not affect the storefront.
+The Magento_Backup allows administrators to perform backups and rollbacks. Types of backups include system, database and media backups. This module relies on the Cron module to schedule backups.
```
Comment (jeff-matthews) [inline app/code/Magento/Backup/README.md]:
```suggestion
The Magento_Backup module allows administrators to perform backups and rollbacks. Types of backups include system, database, and media backups. This module relies on the Cron module to schedule backups.
```

File: app/code/Magento/Backup/README.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+
+The Magento_Backup does not affect the storefront.
```
Comment (jeff-matthews) [inline app/code/Magento/Backup/README.md]:
```suggestion
The Magento_Backup module does not affect the storefront.
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
Minor editorial suggestions

**Review by sidolov — APPROVED** [review]

**Comment by sidolov** [comment]
@diazwatson I have made changes mentioned by @jeff-matthews , will proceed with delivery

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-5850](https://jira.corp.magento.com/browse/ENGCOM-5850) has been created to process this Pull Request

**Review by jeff-matthews — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @jeff-matthews, thank you for the review.
[ENGCOM-5850](https://jira.corp.magento.com/browse/ENGCOM-5850) has been created to process this Pull Request

**Comment by diazwatson** [comment]
Thanks @sidolov @jeff-matthews 👍

**Pull request merged**